build/opt.go

```diff
@@ -32,9 +32,33 @@ import (
 	"github.com/moby/buildkit/util/entitlements"
 	"github.com/opencontainers/go-digest"
 	"github.com/pkg/errors"
+	"github.com/tonistiigi/fsutil"
 )
 
-func toSolveOpt(ctx context.Context, node builder.Node, multiDriver bool, opt Options, bopts gateway.BuildOpts, configDir string, pw progress.Writer, docker *dockerutil.Client) (solveOpt *client.SolveOpt, release func(), err error) {
+type solveOpt struct {
```

Comment:
I'm not quite sure why this wrapper type needs to exist as it doesn't do anything else but embed the other type.

For the method, it could just be a function as well.